src/test/bctest.py

```diff
@@ -11,94 +12,108 @@
 import logging
 
 def parse_output(a, fmt):
-	if fmt == 'json': # json: compare parsed data
-		return json.loads(a)
-	elif fmt == 'hex': # hex: parse and compare binary data
-		return binascii.a2b_hex(a.strip())
-	else:
-		raise NotImplementedError("Don't know how to compare %s" % fmt)
+    """Parse the output a with format fmt. fmt can either be hex or json.
```

Comment:
µNit: no need to mention what fmt can be, when it is mentioned in the code just a line below.